docs/Feature Documentation/Multiple Transports/index.md

```diff
@@ -0,0 +1,82 @@
+# General Description
+
+The Multiple Transports feature allows apps connected to Core to start another connection over a different transport for certain services (for example, an app connected over bluetooth using WiFi as a secondary transport for video streaming). This guide will give an overview of the process which is used to establish a secondary transport connection. See [SDL-0141](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0141-multiple-transports.md) for more details on the original feature proposal.
+
+## Implementation
+- After the proxy is connected to Core, it initiates another connection over a different transport.
+- Core tells the proxy which transport can be used as Secondary Transport.
+- The services that are allowed on the Secondary Transport are specified by Core.
+
+!!! Note
+
+RPC and Hybrid services only run on the Primary transport
+
+!!!
+
+There are three protocol control frames which are used in the implementation of multiple transports
```

Comment:
```suggestion
There are three protocol control frames which are used in the implementation of multiple transports.
```

Reply:
@jacobkeeler it looks like adding the period here was missed in your last commit.